js/views/modals/orderDetail/summaryTab/Fulfilled.js

```diff
@@ -72,11 +72,31 @@ export default class extends BaseVw {
     return this;
   }
 
+  revealEscapeChars(input) {
+    const output = input.replace(/[<>&\n"]/g, x => ({
+      '<': '&amp;lt;',
```

Comment:
Would it be simpler to use encodeURIComponent here?

Reply:
encodeURIComponent is for encoding urls. I'm not sure it's going to convert html to an escaped format like this.

Reply:
It will turn `<a href="http://hackedyou.com">ClickMe</a>` into `%3Ca%20href%3D%22http%3A%2F%2Fhackedyou.com%22%3EClickMe%3C%2Fa%3E`, that'd do what we need wouldn't it?

Reply:
I guess it would but we're using a screw driver where a hammer is needed. It works for that specific case, but I'm wondering if it wouldn't for others we need it to.

Is there some source that suggested this approach?

Reply:
We can leave it as-is.

Reply:
IMO using HTML character entities is significantly more readable than URI encoding. 
```
&lt;a href=&quot;https://www.blocktrail.com/tBTC/tx/7336e170834672538a79ecb293c28957c1d1888ecab8182c3fc7138973e260f2&quot; rel=&quot;nofollow&quot;&gt;the tx test&lt;/a&gt;
```

<a href='https://www.blocktrail.com/tBTC/tx/7336e170834672538a79ecb293c28957c1d1888ecab8182c3fc7138973e260f2' rel='nofollow'>the tx test</a>

I thoroughly explored the approach before deciding to use this, but we can always change it later.